contracts/prize-strategy/PrizeSplit.sol

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.6.12;
+pragma experimental ABIEncoderV2;
+
+import "@openzeppelin/contracts-upgradeable/math/SafeMathUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
+
+/**
+  * @title Abstract prize split contract for adding unique award distribution to static addresses. 
+  * @author Kames Geraghty (PoolTogether Inc)
+*/
+abstract contract PrizeSplit is OwnableUpgradeable {
+  using SafeMathUpgradeable for uint256;
+  
+  /* ============ Variables ============ */
+  PrizeSplitConfig[] internal _prizeSplits;
+
+  /* ============ Enums and Structs ============ */
+  enum TokenType {
+    Ticket,
+    Sponsorship
+  }
```

Comment:
Ah- I guess this hasn't been cut yet?  I remember you mentioning remove the ticket / sponsorship concept entirely.

Probably means you'll want a uint8 for the tokenType

Reply:
Mentioned in discord (could have been more clear) wasn't sure if we wanted to keep an enum type with different labels or just use a uint in the struct instead.

For example...

```
enum TokenType {
    TokenOne,
    TokenTwo
  }
```